utils/switch_task_to_static/docker-compose.yaml

```
@@ -0,0 +1,21 @@
+services:
+  generate_chunks_for_task:
+    image: cvat/server:v2.45.0
```

```suggestion
    image: cvat/server:${CVAT_VERSION:-dev}
```

For compatibility with the dev builds and the base `docker-compose.yml` logic.